### PR TITLE
Use pip3 to pull docker-compose

### PR DESCRIPTION
**Pull request by jsf9k**
Otherwise we get an old version of `docker-compose` that doesn't understand the format of our `docker-compose.yml` files.

Note that this pull request also includes a script for toggling `prevent_destroy` globally.  This is useful in development environments.

### CODE REVIEW

File: packer/docker.json [hunk 1]
```diff
@@ -15,12 +15,7 @@
         },
         "instance_type": "t2.micro",
         "ssh_username": "admin",
-        "ami_name": "cyhy-docker-hvm-{{timestamp}}-x86_64-ebs",
-        "ami_regions": [
-            "us-east-1",
-            "us-west-1",
-            "us-west-2"
-        ],
+        "ami_name": "jsf9k-cyhy-docker-hvm-{{timestamp}}-x86_64-ebs",
```
Comment (dav3r) [inline packer/docker.json]: @jsf9k I don't think you meant to leave this test AMI in here.  Also, don't forget to put the ami_regions back in.

File: packer/nessus.json [hunk 2]
```diff
@@ -15,12 +15,7 @@
         },
         "instance_type": "m4.large",
         "ssh_username": "ec2-user",
-        "ami_name": "cyhy-nessus-hvm-{{timestamp}}-x86_64-ebs",
-        "ami_regions": [
-            "us-east-1",
-            "us-west-1",
-            "us-west-2"
-        ],
+        "ami_name": "jsf9k-cyhy-nessus-hvm-{{timestamp}}-x86_64-ebs",
```
Comment (dav3r) [inline packer/nessus.json]: @jsf9k Ditto here.

File: terraform/scripts/toggle_prevent_destroy [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            ;;
+        \?)
+            echo "Invalid Option: -$OPTARG" 1>&2
+            exit 1
+            ;;
+    esac
+done
+shift $((OPTIND - 1))
+
+find . -maxdepth 1 -name "*.tf" -exec sed -i "s/\(prevent_destroy\s*=\s*\)$before/\1$after/g" {} \;
```
Comment (dav3r) [inline terraform/scripts/toggle_prevent_destroy:28]: Great idea to add this script! 👍

**Review by dav3r — CHANGES_REQUESTED** [review]
See the comments on packer/docker.json and packer/nessus.json for requested corrections.

File: packer/docker.json (hunk 1 above)
Comment (jsf9k) [reply packer/docker.json]: Doh!

**Comment by jsf9k** [comment]
Should be good now.  Thanks for catching that @dav3r!

**Review by dav3r — APPROVED** [review]
All good in the hood

**Pull request merged**